Brokerages/InteractiveBrokers/InteractiveBrokersBrokerage.cs

```diff
@@ -1647,6 +1649,13 @@ public IEnumerable<BaseData> GetNextTicks()
 
             foreach (var tick in ticks)
             {
+                // if we have a cached data time zone for this symbol, perform the conversion 
+                DateTimeZone dataTimeZone;
+                if (_dataTimeZones.TryGetValue(tick.Symbol.Value, out dataTimeZone))
+                {
+                    tick.Time = tick.Time.ConvertTo(TimeZones.NewYork, dataTimeZone);
```

Comment:
The change above is in `IDataQueueHandler.GetNextTicks`, the ticks returned by IB have been previously assigned NY time zone here (with the exception of Forex which is set to EST):
https://github.com/QuantConnect/Lean/blob/master/Brokerages/InteractiveBrokers/InteractiveBrokersBrokerage.cs#L1787
https://github.com/QuantConnect/Lean/blob/master/Brokerages/InteractiveBrokers/InteractiveBrokersBrokerage.cs#L1622

The problem I am trying to solve is that with `AddFuture(symbol, Resolution.Tick)` the algorithm only receives data for futures with NY data time zone:

CL works:
    "Future-usa-CL": {
     "dataTimeZone": "America/New_York",
     "exchangeTimeZone": "America/New_York",

ES doesn't work:
    "Future-usa-ES": {
     "dataTimeZone": "America/Chicago",
     "exchangeTimeZone": "America/Chicago",

Another solution to this problem (without changing code) was to convert all futures market hours to NY time zone, but it didn't seem ideal because usually exchanges publish market hours on their websites in local exchange time, users adding new symbols to the market hours database would need to manually convert the time values to NY time zone (actually not in the case of CME, which reports both time zones on their web site).